orm/test/runtime/org.jboss.tools.hibernate.runtime.v_3_5.test/src/org/jboss/tools/hibernate/runtime/v_3_5/internal/ServiceImplTest.java

```diff
@@ -548,11 +548,18 @@ public void testNewRootClass() {
 	public void testIsInitialized() {
 		assertTrue(service.isInitialized(new Object()));
 	}
-	
-	// TODO JBIDE-28358: reenable this test
-	@Disabled
+
 	@Test
-	public void testGetJPAMappingFilePaths() {
+	public void testGetJPAMappingFilePaths() throws Exception {
```

Comment:
Is this something that is left hanging? The test should not be altered as that would defeat the purpose ;-)